frontend/src/scenes/plugins/install/CustomPlugin.tsx

```diff
@@ -13,7 +13,8 @@ export function CustomPlugin(): JSX.Element {
             <Card>
                 <h3 className="l3">Install Custom Plugin</h3>
                 <p>
-                    To install a third-party or custom plugin, please paste the plugin's repository below.
+                    To install a third-party or custom plugin, paste its GitHub, GitLab or npm URL below. For private
+                    repositories, append <code>?private_token=TOKEN</code> at the end of the URL.
```

Comment:
This should link to what the `TOKEN` should be (i.e.: [GitHub](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token), [GitLab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html), [npm](https://docs.npmjs.com/creating-and-viewing-access-tokens)).